pkg/aws/ec2.go

```diff
@@ -75,6 +75,16 @@ type AssertEC2TagValueInput struct {
 	InstanceID string
 }
 
+// AssertEC2VolumeTagValueInput is used as an input to the AssertEC2VolumeTagValue method.
+type AssertEC2VolumeTagValueInput struct {
+	// The name of the tag to assert exists.
+	TagName string
+	// The value of the tag to assert.
+	Value string
+	// The Volume ID that the method will assert has a tag with the specified tag name and the specified value.
+	VolumeId string
```

Comment:
**Nit:** Idiomatic naming says "ID" should be capitalized.
```suggestion
	VolumeID string
```

Reply:
done